### PR TITLE
Add relationship between RoleEntity & PersonEntity

### PR DESCRIPTION
**Pull request by Vimbayinashe**
The test for verifyEqualsandHashCode has been refactored to include the relationship between the entities ([see Documentation](https://jqno.nl/equalsverifier/errormessages/)).
Closes issue #9

### CODE REVIEW

File: src/main/java/se/iths/librarysystem/entity/RoleEntity.java [hunk 1]
```diff
@@ -14,6 +14,9 @@
     private Long id;
     private String role;
 
+    @OneToMany(mappedBy = "role", cascade = CascadeType.ALL)
```
Comment (ToniKaru) [inline src/main/java/se/iths/librarysystem/entity/RoleEntity.java:17]: FetchType.Lazy?

File: src/test/java/se/iths/librarysystem/entity/GenreEntityTest.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```
Comment (ToniKaru) [inline src/test/java/se/iths/librarysystem/entity/GenreEntityTest.java:8]: This seems to be another issue. Am I missing something?

Comment (Vimbayinashe) [reply src/test/java/se/iths/librarysystem/entity/GenreEntityTest.java:8]: I have merged `main` into this branch 3 times and I think this is a change coming from `main` branch.

**Review by ToniKaru — APPROVED** [review]
Cool

**Pull request merged**